blockchain/block/blockstore.go

```diff
@@ -42,7 +42,6 @@ func (in *Store) ToProto() *iotextypes.BlockStore {
 
 // FromProto converts from proto message
 func (in *Store) FromProto(pb *iotextypes.BlockStore) error {
```

Comment:
still need to make it `FromProto(pb *iotextypes.BlockStore, evmid uint32)` so we can remove the `config.EVMNetworkID()` in line 45

Reply:
or like @CoderZhi  commented, remove it (and fix test)